src/Datasource/EntityInterface.php

```diff
@@ -107,6 +107,12 @@ public function getErrors(): array;
      */
     public function getError(string $field): array;
 
+    /**
+     * Returns all invalids values fields from entity
```

Comment:
See https://github.com/cakephp/cakephp/blob/master/src/Datasource/EntityTrait.php#L1066 for the possible text.
also note the newline.